Source/WebCore/css/CSSPseudoSelectors.json

```diff
@@ -208,7 +208,7 @@
             "comment": "For use in Apple internal apps.",
             "condition": "ENABLE(SERVICE_CONTROLS)",
             "settings-flag": "imageControlsEnabled",
-            "shadow": true
+            "user-agent-shadow-part": true
```

Comment:
Why not use "user-agent-part" to be consistent?

Reply:
Done.